src/libs/ajax/data-table-providers/EntityServiceDataTableProvider.ts

```diff
@@ -40,4 +41,8 @@ export class EntityServiceDataTableProvider implements DataTableProvider {
   deleteTable: DeleteTableFn = async (entityType: string) => {
     return await Ajax().Workspaces.workspace(this.namespace, this.name).deleteEntitiesOfType(entityType)
   }
+
+  downloadTsv: DownloadTsvFn = async (signal: AbortSignal, entityType: string) => {
+    return await Ajax(signal).Workspaces.workspace(this.namespace, this.name).getEntitiesTsv(entityType)
+  }
```

Comment:
This function has parameter types specified two ways (once via DownloadTsvFn and once via AbortSignal/string). Alternatively, it could be written as one of:

```suggestion
  downloadTsv: DownloadTsvFn = async (signal, entityType) => {
    return await Ajax(signal).Workspaces.workspace(this.namespace, this.name).getEntitiesTsv(entityType)
  }
```

```suggestion
  downloadTsv = async (signal: AbortSignal, entityType: string): Promise<Blob> => {
    return await Ajax(signal).Workspaces.workspace(this.namespace, this.name).getEntitiesTsv(entityType)
  }
```

I like the second since the types are more readily apparent.

Reply:
Also could be written more concisely as:

```suggestion
  downloadTsv: DownloadTsvFn = (signal: AbortSignal, entityType: string) => Ajax(signal).Workspaces.workspace(this.namespace, this.name).getEntitiesTsv(entityType)
```

Reply:
my vote is for nick's 2nd suggestion here... the method signature is already enforced by the DataTableProvider interface marked as implemented by the class here.  So the nicest ergonomics is to have the function args and return types inline as in nick's 2nd suggestion.  This is the style I default to myself since many es-linters will complain if those type declarations are missing.